Cargo.toml

```diff
@@ -11,8 +11,15 @@ resolver = "2"
 
 
 [workspace.dependencies]
-bincode = {  version = "2.0.0-rc.3", features = ["serde", "alloc", "derive"], default-features = false }
-serde = { version = "1", features = ["derive","alloc"], default-features = false }
+bincode = { version = "2.0.0-rc.3", features = [
+    "serde",
+    "alloc",
+    "derive",
+], default-features = false }
+serde = { version = "1", features = [
+    "derive",
+    "alloc",
+], default-features = false }
```

Comment:
Can we not have this weird formatting please? I'm guessing this was done automatically without noticiing?

Reply:
github CI formatted it because it wasn't like this in my local repo